polling_stations/apps/pollingstations/tests/test_urls.py

```diff
@@ -27,6 +27,7 @@ def test_trailing_slashes(self):
         )
         urls = [rec["url"] for rec in data]
         for url in urls:
+
             if self.is_exception(url):
                 continue
-            assert url[-1] == "/", url + " does not end with /"
+            assert url[-1] == "/" or ">", url + " does not end with /"
```

Comment:
I wasn't sure about this edit. This assert was failing at /admin/<url> which makes sense, but other than just checking for a trailing < along with / , is there another way to do this?

Reply:
This is the one thing I'm not sure about as well, as I dont know why we would want a URL to end with `>`

Do you know what change caused this test to begin to fail?

Reply:
So I think I figured this out, the URL that is breaking the test is for a view that was added in django 3.2, see the last bullet point here https://docs.djangoproject.com/en/3.2/releases/3.2/#minor-features

I added a `ipdb.set_trace()` in to the test to check the `urls` and that is indeed the only one that doesn't end in the trailing slash.

So I wonder if rather than change the assert to allow `>`, that could lead to us defining other bad urls elsewhere without us realising, we should remove this url from the list of urls to test? So that would mean adding:
```
urls.remove("/admin/<url>")
```
before the for loop.

Reply:
I like Michael's suggestion here: we don't want to add a URL ending in `>` by mistake. 

Another option we might want to think about, especially if we start using `path` with placeholders more, is to allow `/<*>` at the end of a URL 9(e.g, match the opening brace and some value in the middle.

I don't think we need to do that at the moment, but I can imagine that being a valid use case in the future.